addons/website/static/src/components/fullscreen_indication/fullscreen_indication.js

```diff
@@ -24,7 +24,7 @@ export class FullscreenIndication extends Component {
     }
 
     get fullScreenIndicationText() {
-        return markup(sprintf(escape(_t("Press %(key)s to exit full string")), "<span>esc</span>"));
+        return markup(sprintf(escape(_t("Press %(key)s to exit full screen")), "<span>esc</span>"));
```

Comment:
Something is still wrong here 
![image](https://github.com/odoo/odoo/assets/30048408/b989e5c8-bcbf-4032-8d51-95ad0498967a)

Probably just
```suggestion
        markup(sprintf(_t("Press %s to exit full screen"), "<span>esc</span>"))
```
?

Reply:
Okay, that seemed like a mess. Fixed and tested it now.

Reply:
Maybe just
```js
return markup(sprintf(_t("Press %(key)s to exit full screen"), {key: "<span>esc</span>"}));
```
The `escape` doesn't seems needed here at all? 

@odoo/rd-security Can we have your input here

Reply:
Since that would allow any translator to inject malicious code in the translation and `markup` would allow it to just be injected as-is in the HTML. The `escape` makes sure that HTML is properly escaped in the translated string (so they can't add in a `<script>` tag e.g.). In that escaped string, we then inject the `<span>esc</span>`, which is under our control and thus safe.

And the reason why the string is constructed here (and not in the .xml file directly) is because in OWL templates the translations are split per tag. Then we would get 3 unrelated strings to be translated:
- `Press`
- `esc`
- `to exit full screen`
Apart from being hard to translate, that doesn't allow translators to reorder words in other languages etc.